### PR TITLE
sbcl: add head url.

### PR DESCRIPTION
**Pull request by mahiuchun**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/sbcl.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Sbcl < Formula
   url "https://downloads.sourceforge.net/project/sbcl/sbcl/2.0.11/sbcl-2.0.11-source.tar.bz2"
   sha256 "87d2aa53cef092119a1c8b2f3de48d209375a674c3b60e08596838013bd7971d"
   license all_of: [:public_domain, "MIT", "Xerox", "BSD-3-Clause"]
+  head "git://git.code.sf.net/p/sbcl/sbcl", shallow: false
```
Comment (carlocab) [inline Formula/sbcl.rb:7]:
```suggestion
  head "https://git.code.sf.net/p/sbcl/sbcl", shallow: false
```

**Review by carlocab — COMMENTED** [review]
Thanks for your PR, @mahiuchun. One minor suggestion.

**Review by carlocab — APPROVED** [review]

**Comment by carlocab** [comment]
Thanks, @mahiuchun.

File: Formula/sbcl.rb (hunk 1 above)
Comment (mahiuchun) [reply Formula/sbcl.rb]: Turns out `using: :git` must be set when using https. https://github.com/Homebrew/homebrew-core/pull/72090

**Pull request merged**